examples.py/Basics/Color/WaveGradient.py

```diff
@@ -16,19 +16,20 @@ def setup():
 
 def draw():
   frequency = 0
+  j_del = 255.0 / (width + 50)
   for i in range(-75, height + 75):
     # Reset angle to 0, so waves stack properly
     angle = 0
     # Increasing frequency causes more gaps
     frequency += .006
     for j in range(width + 75):
-      py = i + sin(radians(angle)) * amplitude
+      sy = sin(radians(angle))
+      py = i+ int(sy*amplitude)
```

Comment:
Please auto-format.

Reply:
> pep8 format?

Cntrl/Cmmd + T is a Processing IDE "auto-pep8-formatter"